projects/lib/src/chessgame.cpp

```diff
@@ -142,21 +142,21 @@ void ChessGame::stop(bool emitMoveChanged)
 void ChessGame::finish()
 {
 	disconnect(this, SIGNAL(playersReady()), this, SLOT(finish()));
-	for (int i = 0; i < 2; i++)
+	for (auto & i : m_player)
 	{
```

Comment:
Since we're not iterating over indexes anymore I think it would make more sense to have **player** as the auto-variable and maybe change **m_player** -> **m_players** so it reads better. @ilaripih ?